test/audits/manifest/short-name-length.js

```diff
@@ -0,0 +1,59 @@
+/**
+ * Copyright 2016 Google Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+const Audit = require('../../../audits/manifest/short-name-length.js');
+const assert = require('assert');
+const manifestSrc = JSON.stringify(require('./manifest.json'));
+const manifestParser = require('../../../helpers/manifest-parser');
```

Comment:
right now you're not using this manifestParser...

but you should. It's not worth it to handcraft the post-parser objects to pass in: https://github.com/GoogleChrome/lighthouse/blob/master/test/helpers/icons.js#L57-L76